clusterloader2/testing/load/modules/scheduler-throughput.yaml

```diff
@@ -0,0 +1,123 @@
+{{$namespaces := .namespaces}}
+{{$schedulerThroughputNamespaces := .schedulerThroughputNamespaces}}
+{{$schedulerThroughputPodsPerDeployment := .schedulerThroughputPodsPerDeployment}}
+
+{{$ENABLE_HUGE_SERVICES := DefaultParam .CL2_ENABLE_HUGE_SERVICES false}}
+{{$EXIT_AFTER_EXEC := DefaultParam .CL2_EXIT_AFTER_EXEC false}}
```

Comment:
If you implement idea with actions then we could get rid of this flag and move it to config.yaml.

Reply:
Done